docs/administrator/verify-artifacts.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Use cosign to verify signed images
+---
+
+Karmada has introduced cosign to verify the released images since version v1.6.1. The specific operation is as follows：
```

Comment:
```suggestion
Karmada has introduced cosign to verify the released images since version v1.6. The specific operation is as follows：
```

Reply:
> 

sorry，I may not get it. I  do't find we support v1.6.0.  In fact I was failed when I verify v1.6.0

Reply:
Sorry, my bad. 
#3434 was merged on Jun 16, it's release-1.7.  I think we didn't sign the image for v1.6.x.

For now, people can only verify the images against the `latest` image, am I right?

Reply:
> Sorry, my bad. #3434 was merged on Jun 16, it's release-1.7. I think we didn't sign the image for v1.6.x.
> 
> For now, people can only verify the images against the `latest` image, am I right?

Yes,  you are right. I make an mistake here also. I will change version to `v.1.7`.